process-services/latest/support/index.md

```diff
@@ -38,6 +36,7 @@ our Support team - submit a case via [Hyland Community]({% link support/latest/c
 | Jetty 9.4 | |
 | | |
 | **JDKs** | |
+| OpenJDK 17 | |
```

Comment:
As of now no changes have been added to support JAVA 17.

Reply:
So this shouldnt be added.

Reply:
I edited the file to remove Java 17.